patches/minecraft/net/minecraft/world/level/block/RedStoneWireBlock.java.patch

```diff
@@ -0,0 +1,39 @@
+--- a/net/minecraft/world/level/block/RedStoneWireBlock.java
++++ b/net/minecraft/world/level/block/RedStoneWireBlock.java
+@@ -199,7 +_,8 @@
+       BlockState blockstate = p_55523_.m_8055_(blockpos);
+       if (p_55526_) {
+          boolean flag = this.m_55612_(p_55523_, blockpos, blockstate);
+-         if (flag && m_55640_(p_55523_.m_8055_(blockpos.m_7494_()))) {
++         flag &= p_55523_.m_8055_(blockpos.m_7494_()).canRedstoneConnectTo(p_55523_, blockpos.m_7494_(), null);
++         if (flag) {
+             if (blockstate.m_60783_(p_55523_, blockpos, p_55525_.m_122424_())) {
+                return RedstoneSide.UP;
+             }
+@@ -207,8 +_,9 @@
+             return RedstoneSide.SIDE;
+          }
+       }
+-
```

Comment:
Doesn't really matter, but you delete an empty line here for no reason which means the hunk is one line larger.